public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -103,6 +108,99 @@ export const DataConnection = (props: any) => {
     },
   ];
 
+  const QueryOrAccelerateData = () => {
+    console.log(chrome);
+    switch (datasourceDetails.connector) {
+      case 'S3GLUE':
+        return (
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiCard
+                icon={<EuiIcon size="xxl" type="discoverApp" />}
+                title={'Query data'}
+                description="Query your data in Data Explorer or Observability Logs."
+                onClick={() => application!.navigateToApp(observabilityLogsID)}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiCard
+                icon={<EuiIcon size="xxl" type="bolt" />}
+                title={'Accelerate performance'}
+                description="Accelerate performance through OpenSearch indexing."
+                onClick={() => {}}
```

Comment:
@sumukhswamy @ps48 still not sure what this card is supposed to do, since acceleration is no longer another page?

Reply:
This will redirect to query workbench with accelerate table flyout open. Can keep it blank for now.

Reply:
Will keep it blank for now, when endpoint is ready will update it separately